This module is not required on GNU/Linux

--- a/is-absolute/index.js
+++ b/is-absolute/index.js
@@ -1,7 +1,7 @@
 'use strict';
 
 var isRelative = require('is-relative');
-var isWindows = require('is-windows');
+var isWindows = false;
 
 /**
  * Expose `isAbsolute`
@@ -20,7 +20,7 @@
   if (typeof fp !== 'string') {
     throw new TypeError('isAbsolute expects a string.');
   }
-  return isWindows() ? isAbsolute.win32(fp) : isAbsolute.posix(fp);
+  return isAbsolute.posix(fp);
 }
 
 /**
